microceph/ceph/services.go

```diff
@@ -83,6 +100,26 @@ func RestartCephService(service string) error {
 	return nil
 }
 
+func getUpRgws() (common.Set, error) {
+	// check if rgw was up for atleast 2 seconds.
+	rgwSocketFiles := common.FilterFilesInDir(constants.RgwSockPatter, constants.GetPathConst().RunPath)
+	for _, file := range rgwSocketFiles {
+		age := common.GetFileAge(file)
```

Comment:
Looking at the `GetFileAge` function, it appears that a return value of 0 is an error, isn't it?

Reply:
IMHO it might be ok not to handle the `0` error case here explicitly, we're logging it in GetFileAge itself, and just assume this rgw service is down. However might want to add a comment to document this decision